src/nvim/testdir/test_normal.vim

```diff
@@ -431,6 +433,7 @@ func! Test_normal13_help()
 endfunc
 
 func! Test_normal14_page()
+  throw "skipped: Nvim regression: CTRL-F with 'scrolloff'"
```

Comment:
This is a legitimate regression. We changed the order of when the cursor is placed in a window vs adjusting for 'scrolloff'.